### DISCUSSION TITLE
HBO

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#393] [Show and tell]
### Channel Name

HBO

### Channel ID

HBOEast.us

### Stream URL

http://190.171.101.36/HBO/tracks-v3a1/mono.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by TKList** [comment]
http://190.171.101.36/HBO/index.m3u8

**Comment by spotanjo3** [comment]
This is Spanish, I believe. What about USA version in English ?

**Comment by TKList** [comment]
Columbia or Caribbean

**Comment by spotanjo3** [comment]
Huh? I am sorry but I dont understand. ☺️😇

**Comment by TKList** [comment]
They probably don't allow US HBO on here.

**Comment by spotanjo3** [comment]
I am afraid so… BUT at least maybe you can try ? 😎

**Comment by BellezaEmporium** [comment]
IP doesn't answer.

**Comment by spotanjo3** [comment]
What does that mean?

**Comment by BellezaEmporium** [comment]
That I have no access to the resource.

**Comment by spotanjo3** [comment]
Oh ok. 😇

**Comment by BellezaEmporium** [comment]
OK, weirdly enough it works, US VPN being used. Forced subtitles in Spanish if people speak in English.

EDIT : OK, it only works if the IP is based in the US it seems, rather odd for a Flussonic server.

**Comment by spotanjo3** [comment]
Post it if its in English and closed captioning in English. :)

**Comment by spotanjo3** [comment]
And maybe you can solve the problem for SYFY also?

**Comment by BellezaEmporium** [comment]
I don't have any english links for SyFy sadly.

**Comment by spotanjo3** [comment]
I understand but….. How did they manage to get an English version then? 

What about an English version of HBO?

**Comment by freearhey** [comment]
http://190.171.101.36/HBO/tracks-v3a1/mono.m3u8 (offline)
<img width="367" alt="image" src="https://user-images.githubusercontent.com/7253922/232229452-4f033015-3727-4687-a328-d17807c64fac.png">

http://190.171.101.36/HBO/index.m3u8 (offline)
<img width="277" alt="image" src="https://user-images.githubusercontent.com/7253922/232229477-25c35142-a286-4999-943b-28d598764d43.png">
